magefile.go

```diff
@@ -32,7 +32,7 @@ var (
 
 	publicDir    = "./public"
 	buildDir     = "./build"
-	packagePaths = []string{"./dev/packages/alpha/", "./dev/packages/example/", "./dev/packages/beats/"}
```

Comment:
Why did you remove the example ones too? I would keep them in as default but I opened https://github.com/elastic/package-registry/pull/435 to make it more flexible.

Reply:
Fixed.